backend/config/settings.py

```diff
@@ -0,0 +1,160 @@
+    'django.contrib.staticfiles',
+    'corsheaders',
+    'rest_framework',
+    'oauth_pda_app',
+    'backend.pumpkin',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
+]
+
+
+ROOT_URLCONF = 'backend.config.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'backend.config.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/4.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/4.1/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
+    },
+]
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/4.1/topics/i18n/
+
+LANGUAGE_CODE = 'en-us'
+
+TIME_ZONE = 'UTC'
+
+USE_I18N = True
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/4.1/howto/static-files/
+
+STATIC_URL = 'static/'
+
+# Default primary key field type
+# https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
+
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+CORS_ORIGIN_WHITELIST = [
+     'http://localhost:3000'
+]
+
+
+
+OAUTH_SETTINGS = {
+    'client_id': CLIENT_ID,                                     # ID du client OAuth
+    'client_secret': CLIENT_SECRET,                             # Secret du client OAuth
+    'authorization_url': AUTHORIZATION_URL,                     # lien d'autorisation OAuth sur le PDA
+    'token_url': TOKEN_URL,                                     # lien de récupération des tokens sur le PDA
+    'redirect_uri': REDIRECT_URI,                               # adresse de callback
```

Comment:
```suggestion
    'client_id': os.environ['CLIENT_ID'], # ID du client OAuth
    'client_secret': os.environ['CLIENT_SECRET'], # Secret du client OAuth
    'authorization_url': os.environ['AUTHORIZATION_URL'], # lien d'autorisation OAuth sur le PDA
    'token_url': os.environ['TOKEN_URL'], # lien de récupération des tokens sur le PDA
    'redirect_uri': os.environ['REDIRECT_URI'], # adresse de callback
```

Ne mets qu'un espace avant tes inline comments (c'est conventionnel)
Et ne créé pas de variable qui ne servent qu'a donner une valeur a une autre variable. Lis directement le env ici.